dir.props

```diff
@@ -504,15 +504,15 @@
   <!-- Set up the default output and intermediate paths -->
   <PropertyGroup>
     <OSPlatformConfig>$(OSGroup).$(Platform).$(ConfigurationGroup)</OSPlatformConfig>
-    <TargetOutputRelPath Condition="'$(TargetGroup)'!=''">$(TargetGroup)/</TargetOutputRelPath>
+    <TargetOutputExtension Condition="'$(TargetGroup)'!=''">.$(TargetGroup)</TargetOutputExtension>
```

Comment:
As we chatted lets keep these in a sub-dir but for TargetGroup empty use netstandard as the folder name.